src/browser/Navigators/Navigator/Input.js

```diff
@@ -197,6 +197,26 @@ export const update =
     }
   };
 
+const change =
+  (model, value, selection) => {
+    const [edit, editFX] =
+      Editable.update(model, Editable.Change(value, selection));
+
+    const fx =
+      ( model.value.includes(edit.value)
```

Comment:
> Wouldn't this be edit.value.includes(model.value)? If edit.value is currently "hell" and the model was previously "hello", etc., unless I got these values backwards

So if `edit.value === "hell"` and `model.value === "hello"` this means that `o` was deleted. Theres for `model.value.includes(edit.value) === true` since `"hello".includes("hell")`.

Does that makes sense ?